plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -406,7 +406,10 @@ export const createSessionReplayEvent = (
 ) => {
     const timestamps = events
         .filter((e) => !!e?.timestamp)
-        .map((e) => castTimestampOrNow(DateTime.fromMillis(e.timestamp), TimestampFormat.ClickHouse))
+        .filter((e) => Math.sign(e.timestamp) === 1)
+        .map((e) => DateTime.fromMillis(e.timestamp))
```

Comment:
So we'd just call `DateTime.fromMillis(e?.timestamp)` and then check validity

Arguably, then we should have tests on this specifically to simplify the tests but 🤷